test-variables.yml.example

```diff
@@ -4,30 +4,7 @@ BROWSER:
   # --disable-dev-shm-usage  and --no-sandbox are required for running chromedriver in a container
   OPTIONS: add_argument("--ignore-certificate-errors");add_argument("window-size=1920,1024");add_argument("--disable-dev-shm-usage");add_argument("--no-sandbox")
 OCP_CONSOLE_URL: "http://console-openshift-console.apps.my-cluster.test.redhat.com/"
-ODH_DASHBOARD_URL: "http://odh-dashboard-opendatahub.apps.my-cluster.test.redhat.com/"
-RHODS_PROMETHEUS_URL: "https://prometheus-redhat-ods-monitoring.apps.my-cluster.test.redhat.com/"
-# RHODS_PROMETHEUS_TOKEN can be obtained running: oc serviceaccounts get-token prometheus -n redhat-ods-monitoring #
-RHODS_PROMETHEUS_TOKEN: "prometheus-token"
-TEST_USER:
-  AUTH_TYPE: foo-auth
-  USERNAME: foo-user
-  PASSWORD: foo-passwd
 OCP_ADMIN_USER:
   AUTH_TYPE: "kube:admin"
```

Comment:
Why is it like kube;admin and not simply kubeadmin?

Reply:
This is a template, where the actual AUTH TYPE used would be replaced. so kube:admin is just a dummy value.